lib/vehicle_factory.rb

```diff
@@ -0,0 +1,20 @@
+class VehicleFactory
+  attr_reader :vehicles
+
+  def initialize
+    @vehicles = []
+  end
+
+  def create_wa_vehicles(dmv_data)
```

Comment:
In the Iteration 2 interaction pattern, this method was called `create_vehicles`. Not a huge deal for this submission, but if we ever wanted `VehicleFactory` to handle data sets from other states, it might need to be renamed.